test/suite/test_bf32981.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+#
```

Comment:
We have a category called `test_bug###.py` that would be preferable naming. The reason is that then you can `runpy bug` and it will be run. There are no existing tests with a bf prefix. This one would be `test_bug033.py`.

Reply:
Done.
Renamed to test_bug033.py and changed comment identifying the file.